From 368be2e8a4290da9e49c5625ed207885d345cfcf Mon Sep 17 00:00:00 2001
From: Holger Freyther <zecke@openmoko.org>
Date: Tue, 13 May 2008 18:53:40 +0100
Subject: [PATCH] [power] Use the bq27000 battery to provide charging values for apm
     Built the apm emulation for gta02 and say that the gta02 battery
     is supposed to be used for this apm emulation.

Signed-Off-By: Holger Freyther <zecke@openmoko.org>
---
 defconfig-gta02                 |    2 +-
 drivers/power/bq27000_battery.c |    1 +
 2 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/defconfig-gta02 b/defconfig-gta02
index 1fa2d80..c4947b7 100644
--- a/defconfig-gta02
+++ b/defconfig-gta02
@@ -1018,7 +1018,7 @@ CONFIG_SPI_S3C24XX_GPIO=y
 CONFIG_POWER_SUPPLY=y
 # CONFIG_POWER_SUPPLY_DEBUG is not set
 # CONFIG_PDA_POWER is not set
-# CONFIG_APM_POWER is not set
+CONFIG_APM_POWER=y
 # CONFIG_BATTERY_DS2760 is not set
 CONFIG_BATTERY_BQ27000_HDQ=y
 CONFIG_GTA02_HDQ=y
diff --git a/drivers/power/bq27000_battery.c b/drivers/power/bq27000_battery.c
index d19186a..cf57dcf 100644
--- a/drivers/power/bq27000_battery.c
+++ b/drivers/power/bq27000_battery.c
@@ -299,6 +299,7 @@ static int bq27000_battery_probe(struct platform_device *pdev)
 	di->bat.get_property   = bq27000_battery_get_property;
 	di->bat.external_power_changed =
 				  bq27000_battery_external_power_changed;
+	di->bat.use_for_apm = 1;
 	di->hdq_read = pdata->hdq_read;
 	di->hdq_write = pdata->hdq_write;
 	di->rsense_mohms = pdata->rsense_mohms;
-- 
1.5.6.5

